visualization/vis-master/src/store/data.js

```diff
@@ -8,6 +8,8 @@ import latestDistData from '!json!../assets/data/distributions/season-latest-nat
 
 import incCaseSeasonData from '!json!../assets/data/season-Incident Cases.json'
 
+import incDeathSeasonData from '!json!../assets/data/season-Incident Deaths.json'
```

Comment:
Not important, but if we're loading this by default, can we remove the previous line (the one importing the inc cases data)?